Automattic/newspack-blocks
# fix: update newspack-components dep version

### All Submissions:






### Changes proposed in this Pull Request:



Fixes a build error due to a missing dependency in Newspack Components.

### How to test the changes in this Pull Request:

Test `npm ci && npm run build`. Also, all CI jobs below should be passing. 👇 No functional changes.

### Other information:
